etc/config/c++.amazon.properties

```diff
@@ -157,6 +157,10 @@ compiler.clang_p1144.exe=/opt/compiler-explorer/clang-relocatable-trunk/bin/clan
 compiler.clang_p1144.semver=(experimental P1144)
 compiler.clang_p1144.options=-std=c++2a -stdlib=libc++
 compiler.clang_p1144.notification=Experimental __is_trivially_relocatable; see <a href="https://quuxplusone.github.io/draft/d1144.html" target="_blank" rel="noopener noreferrer">P1144 <sup><small class="glyphicon glyphicon-new-window opens-new-window" title="Opens in a new window"></small></sup></a>
+compiler.clang_autonsdmi.exe=/opt/compiler-explorer/clang-autonsdmi-trunk/bin/clang++
+compiler.clang_autonsdm.semver=(experimental auto NSDMI)
+compiler.clang_autonsdm.options=-std=c++2a -stdlib=libc++
+compiler.clang_autonsdm.notification=Experimental auto NSDMI;
```

Comment:
You have a trailing semicolon here, can you remove? Maybe make a link to a website explaining what this is too? (Like the p1144 one?) If not the website at least drop the semicolon please :)

Reply:
Fixed, sorry for that.
However, providing a website will be a bit difficult - it could be my blog (and advertising my blog on CE seems wrong !) - or an old paper https://wg21.link/n2713
What do you think?

Reply:
I'm happy to leave it blank, and then update it when you have a link to an article. Linking to your blog seems fine to me if it explains the reason for the compiler (and what NSDMI is :))